src/libs/actions/Report.ts

```diff
@@ -1322,18 +1330,35 @@ function handleReportChanged(report: OnyxEntry<Report>) {
         let callback = () => {
             Onyx.set(`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`, null);
```

Comment:
Don't we need to clear the `REPORT_DRAFT_COMMENT` of `report.reportID` as well?

Reply:
Thank you very much @dominictb, I'll continue testing, I haven't tested it in native yet, I'll come back with the checklist ASAP.